rotkehlchen/chain/evm/decoding/spark/constants.py

```diff
@@ -0,0 +1,13 @@
+from typing import Final
+
+from rotkehlchen.chain.evm.decoding.types import CounterpartyDetails
+
+CPT_SPARK: Final = 'spark'
+
+SPARK_LABEL: Final = 'Spark'
+
+SPARK_COUNTERPARTY_DETAILS = CounterpartyDetails(
+    identifier=CPT_SPARK,
+    label=SPARK_LABEL,
```

Comment:
question: do you need to create a variable for label? You can always access SPARK_COUNTERPARTY_DETAILS.label that is cleaner